knowledge-base/repair-simple-cross-reference-table.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Repair Simple Cross-Reference Table 
+description: Repair Simple Cross-Reference Table
+type: how-to
+page_title: Repair Simple Cross-Reference Table
+slug: repair-simple-cross-reference-table
+position: 0
+tags: 
+res_type: kb
+---
+
+<table>
+<thead>
+	<tr>
+		<th>Product Version</th>
+		<th>Product</th>
+		<th>Author</th>
+	</tr>
+</thead>
+<tbody>
+	<tr>
+		<td>2020.1.316</td>
+		<td>RadPdfProcessing</td>
+		<td><a href="https://www.telerik.com/blogs/author/martin-velikov">Martin Velikov</a></td>
+	</tr>
+</tbody>
+</table>
```

Comment:
The other KBs have a different structure here.

Reply:
Both are valid and produce the same result.